test/build/alpine/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:12.20.0-alpine
+
+RUN mkdir /hoprd
+WORKDIR /hoprd
+RUN npm install --unsafe-perm=true -g @hoprnet/hoprd@v1.54.5
```

Comment:
Seems like the release is hardcoded?

Reply:
This is just for debug by the avado guys